backend/src/test/java/de/neuefische/cgnjava222/productgallery/service/FileIntegrationTest.java

```diff
@@ -59,40 +71,65 @@ void uploadFile() throws Exception {
                 )
         ).thenReturn(Map.of("url", "hi", "public_id", "bla"));
 
-        MockMvc mockMvc
-                = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
-        mockMvc.perform(multipart("/api/image/uploadFile/").file(firstFile))
+        mockMvc.perform(multipart("/api/image/uploadFile/").file(firstFile).with(csrf()))
                 .andExpect(status().isCreated());
     }
 
+    // @Test
+    // @WithMockUser(username = "frank", authorities = {"ADMIN", "USER"})
+    // void uploadFileExceptionsFileNameNull() {
+    //     MockMultipartFile nullNamedFile = new MockMultipartFile(
+    //             "file", null,
+    //             MediaType.TEXT_PLAIN_VALUE,
+    //             "Hello, World!".getBytes());
+
+    //     Exception exception = assertThrows(Exception.class, () -> {
+    //         mockMvc.perform(multipart("/api/image/uploadFile/").file(nullNamedFile).with(csrf()));
+    //     });
+
+    //     String expectedMessage = "File Upload der Datei: wurde nicht durchgeführt";
+    //     String actualMessage = exception.getMessage();
+
+    //     assertThat(actualMessage).contains(expectedMessage);
+    // }
```

Comment:
Please delete, pushing commented out code to main is not a good practice. Keep you code base clean! ;)